### DISCUSSION TITLE
Material Texture Inputs?

### DISCUSSION THREAD

**Discussion by ninarular** [cwapi3d/cwapi3dpython#186] [Q&A] [answered]
Hey Guys! 

I was working on a plugin to export a CSV list of CLT materials to the internal materials list, and wanted to export the material color (texture?) as well to match the general template of our already used materials. When looking into this I saw that the input for the material texture input hasnt been defined :( Does anyone know what a0 and a1 are referring to? I'm assuming a0 is the material ID. 

![image](https://github.com/cwapi3d/cwapi3dpython/assets/110792376/fb368140-defb-4786-9be6-2a86e264db81)

**Comment by Brunner246** [comment, marked as answer]
a0 = material_id
a1 = color_number

**Reply by ninarular** [reply]
Thank you!
